specs/023-pwa-push-notifications/prd.md

```
@@ -0,0 +1,388 @@
+{
+  "gcm_sender_id": "103953800507"
+}
+```
+
+> Note: `gcm_sender_id` with the value `103953800507` is required for Chrome/FCM compatibility regardless of using VAPID. This is a well-known constant.
+
+### 5.8 Internationalization
+
+Add translation keys to `src/core/i18n/en/translation.en.json`:
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)